### DISCUSSION TITLE
Enable Github Sponsor

### DISCUSSION THREAD

**Discussion by Lunik** [BerriAI/litellm#2750] [General]
I would like to contribute to the project through [Github Spronsor](https://github.com/sponsors).

**Comment by ishaan-jaff** [comment]
<img width="358" alt="Screenshot 2024-03-30 at 8 41 59 AM" src="https://github.com/BerriAI/litellm/assets/29436595/a84a4ad6-e79d-469e-b690-f63ba6f01e6e">

enabled already

**Reply by Lunik** [reply]
I see but the issue is that it is through a third party platform and the amount is not customisable.

**Comment by hughcrt** [comment]
Hi @ishaan-jaff ,

I'm a bit confused by the usage of stripe for sponsoring. Is it true sponsoring, or does it give access to paid features?
We'd love to sponsor your project, but it would be great if it would be via Github built-in sponsoring system.
